packages/@react-spectrum/tooltip/src/TooltipTrigger.tsx

```diff
@@ -21,6 +21,9 @@ import {useOverlayPosition} from '@react-aria/overlays';
 import {useTooltipTrigger} from '@react-aria/tooltip';
 import {useTooltipTriggerState} from '@react-stately/tooltip';
 
+const DEFAULT_OFFSET = 7;
```

Comment:
where did this 7 come from?

Reply:
Looks like a magic number :) It was the closest offset that visually matches Spectrum's mocks.